cmd/sql/flow.go

```diff
@@ -128,7 +128,7 @@ func getProjectDirFromArgs(args []string) string {
 
 // Read config cmd output for retrieving config settings such as airflow_home
 func readConfigCmdOutput(key string) (string, error) {
-	args := []string{key}
+	args := []string{"get", key}
```

Comment:
Thanks @neel-astro for the review. That is very good question. **Right now, everything we add here should not be included in astro-cli releases until sql-cli 0.3.0 is released.** For bug fixes regarding the current release (0.2.x), we will notify you and would ask you to cherry-pick those commits. Sorry for the circumstances this may cause to you. We are planning to [release 0.3.0](https://github.com/astronomer/astro-sdk/milestone/19) on 16th of January.